delirium/apps.py

```diff
@@ -3,32 +3,32 @@
 from .const import *
 from .dns import FakeDNSServer
 
+
 @click.group()
 def delirium():
     pass
 
+
 @click.command()
-@click.option('--listen', default=DEFAULT_LISTEN_ADDR, help='Address to listen on')
-@click.option('--port', default=DEFAULT_LISTEN_PORT, type=int, help='Port to listen on (UDP)')
-@click.option('--time', default=DEFAULT_CACHE_DURATION, type=int, help='Seconds for which cache entries should exist')
-@click.option('--range', default=DEFAULT_ADDR_RANGE, help='Range of IP addresses to randomly generate')
-@click.option('--cache_type', default=CACHE_TYPE.DICTIONARY, help='Cache type.')
-@click.option('--db-path', help='Path to sqlite3 database')
-def dns(listen, port, time, range, cache_type, db_path):
+@click.option('-l','--laddr', default=DEFAULT_LISTEN_ADDR, help='Address to listen on')
+@click.option('-p','--lport', default=DEFAULT_LISTEN_PORT, type=int, help='Port to listen on (UDP)')
+@click.option('-t','--time', default=DEFAULT_CACHE_DURATION, type=int, help='Seconds for which cache entries should exist')
+@click.option('-a','--addr-pool', default=DEFAULT_ADDR_RANGE, help='Range of IP addresses to randomly generate')
+@click.option('-d','--db-path', default=DEFAULT_DB_PATH ,help='Path to sqlite3 database')
+def dns(laddr, lport, time, addr_pool, db_path):
+    cache_type = CACHE_TYPE.DATABASE
```

Comment:
Instead of storing this constant as a variable, go ahead and just use the constant directly in the instantiation of the `FakeDNSServer` object: `s = FakeDNSServer(laddr, lport, time, addr_pool, CACHE_TYPE.DATABASE, db_path)`